docs/glossary.md

```diff
@@ -74,187 +215,135 @@ license: "This software is licensed under the Apache License version 2."
+- A cloud-based registry service for Docker containers.
+  Users can publish, browse and download containerized software using this service.
 
 ## Docker image
 
-- A lightweight, standalone, and executable package that includes everything needed to run a piece of software, including the code, runtime, libraries, and system tools.
+- A lightweight, standalone package that includes everything needed to run a software.
+  A Docker image typically comprises the code, runtime, libraries, and system tools.
 
-## Docker file
+## Dockerfile
 
-- A configuration file used to create a Docker image, specifying the base image, dependencies, and commands for building the image.
+- A configuration file for building a Docker image.
+  A Dockerfile contains instructions to build a standard image for distributing a software.
 
-## Dropwizard Metrics
+## Dropwizard metrics
 
 - A Java library for measuring the performance of applications and providing support for various metric types.
 
-## Amazon Elastic Block Store (EBS)
-
-- A scalable block storage service provided by Amazon Web Services.
-
 ## Environment variables
 
-- Variables used to pass information to running processes.
+- Variables used to customize the runtime configuration for a process.
 
 ## Geo-distributed
 
 - The distribution of data or services across multiple geographic locations.
 
+## Git
+
+- A distributed version control system used for tracking software artifacts.
+
 ## GitHub
 
-- A web-based platform for version control and collaboration using Git.
+- A web-based platform for version control and community collaboration using Git.
 
 ## GitHub Actions
 
-- A continuous integration and continuous deployment (CI/CD) service provided by GitHub, used for automating build, test, and deployment workflows.
+- A continuous integration and continuous deployment (CI/CD) service provided by GitHub.
+  GitHub Actions are used for automating the build, test, and deployment workflows.
 
 ## GitHub labels
 
-- Tags assigned to GitHub issues or pull requests for organization, categorization, or workflow automation.
+- Labels assigned to GitHub issues or pull requests for organization or workflow automation.
 
 ## GitHub pull request
 
-- A proposed change to a repository submitted by a user through the GitHub platform.
+- A proposed change to a GitHub repository submitted by a user.
 
 ## GitHub repository
 
 - The location where GitHub stores a project's source code and related files.
 
 ## GitHub workflow
 
-- A series of automated steps defined in a YAML file that runs in response to events on a GitHub repository.
-
-## Git
-
-- A version control system used for tracking changes and collaborating on source code.
-
-## GPG/GnuPG
-
-- Gnu Privacy Guard or GnuPG, an open-source implementation of the OpenPGP standard, used for encrypting and signing files and emails.
+- A series of automated steps that are triggered by certain events on a GitHub repository.
```

Comment:
Suggestion: "A series of automated steps triggered by specific events on a GitHub repository."